src/components/Navbar2/Navbar2.tsx

```diff
@@ -0,0 +1,487 @@
+import React, { useState, useCallback } from 'react'
```

Comment:
There seems to be an issue with the color of the menus when they're not hovered. They're too dark, which makes it difficult to read for people with visual problems. This is an accessibility issue that should be fixed before merging the PR.

How it's now:
<img width="1194" alt="Screenshot 2023-12-29 at 12 45 11" src="https://github.com/decentraland/ui/assets/1120791/7140790b-2849-4f80-9f25-8136f2efc015">
How it should be (coming from the Figma):
<img width="590" alt="Screenshot 2023-12-29 at 12 45 20" src="https://github.com/decentraland/ui/assets/1120791/43ffda29-8b12-4c99-9cff-eca2a9538f80">

Reply:
the colors you are taking from figma are the hover one.
You can see the proper colors here in figma

<img width="1064" alt="Screenshot 2023-12-29 at 15 57 50" src="https://github.com/decentraland/ui/assets/6572776/b152c8c7-cb3f-48f6-bde1-1bce61753bb9">